index.js

```diff
@@ -60,7 +60,11 @@ const getBorderChars = borderStyle => {
 	return chars;
 };
 
-const getBackgroundColorName = x => camelCase('bg', x);
+// Color must either be a valid chalk color or a color hex ( #55aabb )
```

Comment:
@ndelangen What about use [chalk-pipe](https://github.com/LitoMore/chalk-pipe) to solve colors?

We can just pass a string to get colors, and even more styles.

Reply:
What other styles would make sense?

Reply:
Combined style and CSS keyword styles:

```
#55aabb.bold
lightgrey
pink
```

> See doc here: https://github.com/LitoMore/chalk-pipe#valid-styles

Reply:
@LitoMore I don't want to overcomplicate it.